vnet/spec/vnet/endpoints/1.0/routes_spec.rb

```diff
@@ -38,18 +38,9 @@ def app
   end
 
   describe "PUT /:uuid" do
-    let!(:new_interface) { Fabricate(:interface) { uuid 'if-newif' } }
-    let!(:new_network) { Fabricate(:network) { uuid "nw-newnw" } }
-    let!(:route_link) { Fabricate(:route_link) { uuid "rl-newroute" } }
-
     accepted_params = {
-      :interface_uuid => "if-newif",
-      :network_uuid => "nw-newnw",
-      :route_link_uuid => "rl-newroute",
-      :ipv4_network => "192.168.3.0",
-      :ipv4_prefix => 16,
```

Comment:
Same comment as for `route_links`. Now that the PUT request no longer accepts any parameters, it doesn't do anything any more so I think we should remove it.